### ISSUE TITLE
issue in Pandas_Seaborn.ipynb with holoviews 1.4.1 and seaborn master of 2015-11-20

### ISSUE THREAD

**Issue by stonebig** [ioam/holoviews#388]
````
%%opts Distribution (rug=True kde_kws={'color':'indianred','linestyle':'--'})
hv.Distribution(np.random.randn(10), kdims=['Activity'])
````

gives this:
````
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-4-578a4209e335> in <module>()
----> 1 hv.Distribution(np.random.randn(10), kdims=['Activity'])

D:\WinPython\basedir34\buildQt5\winpython-3.4.4.amd64\python-3.4.4.amd64\lib\site-packages\holoviews\core\data.py in __init__(self, data, **kwargs)
     67                                              datatype=kwargs.get('datatype'))
     68         (data, kdims, vdims, self.interface) = initialized
---> 69         super(Columns, self).__init__(data, **dict(kwargs, kdims=kdims, vdims=vdims))
     70         self.interface.validate(self)
     71 

D:\WinPython\basedir34\buildQt5\winpython-3.4.4.amd64\python-3.4.4.amd64\lib\site-packages\holoviews\core\dimension.py in __init__(self, data, **params)
    542                                   for d in params.pop(group)]
    543                 params[group] = dimensions
--> 544         super(Dimensioned, self).__init__(data, **params)
    545         self.ndims = len(self.kdims)
    546         cdims = [(d.name, val) for d, val in self.cdims.items()]

D:\WinPython\basedir34\buildQt5\winpython-3.4.4.amd64\python-3.4.4.amd64\lib\site-packages\holoviews\core\dimension.py in __init__(self, data, id, **params)
    251             params['group'] = long_name
    252 
--> 253         super(LabelledData, self).__init__(**params)
    254         if not group_sanitizer.allowable(self.group):
    255             raise ValueError("Supplied group %r contains invalid characters." %

D:\WinPython\basedir34\buildQt5\winpython-3.4.4.amd64\python-3.4.4.amd64\lib\site-packages\param\parameterized.py in __init__(self, **params)
   1019         self.__generate_name()
   1020 
-> 1021         self._setup_params(**params)
   1022         object_count += 1
   1023 

D:\WinPython\basedir34\buildQt5\winpython-3.4.4.amd64\python-3.4.4.amd64\lib\site-packages\param\parameterized.py in override_initialization(parameterized_instance, *args, **kw)
    956         original_initialized=parameterized_instance.initialized
    957         parameterized_instance.initialized=False
--> 958         fn(parameterized_instance,*args,**kw)
    959         parameterized_instance.initialized=original_initialized
    960     return override_initialization

D:\WinPython\basedir34\buildQt5\winpython-3.4.4.amd64\python-3.4.4.amd64\lib\site-packages\param\parameterized.py in _setup_params(self, **params)
   1409                 self.warning("Setting non-parameter attribute %s=%s using a mechanism intended only for parameters",name,val)
   1410             # i.e. if not desc it's setting an attribute in __dict__, not a Parameter
-> 1411             setattr(self,name,val)
   1412 
   1413 

D:\WinPython\basedir34\buildQt5\winpython-3.4.4.amd64\python-3.4.4.amd64\lib\site-packages\param\__init__.py in __set__(self, obj, val)
   1070     def __set__(self,obj,val):
   1071         """Set to the given value, raising an exception if out of bounds."""
-> 1072         self._check_bounds(val)
   1073         super(List,self).__set__(obj,val)
   1074 

D:\WinPython\basedir34\buildQt5\winpython-3.4.4.amd64\python-3.4.4.amd64\lib\site-packages\param\__init__.py in _check_bounds(self, val)
   1089             if min_length is not None and max_length is not None:
   1090                 if not (min_length <= l <= max_length):
-> 1091                     raise ValueError("%s: list length must be between %s and %s (inclusive)"%(self._attrib_name,min_length,max_length))
   1092             elif min_length is not None:
   1093                 if not min_length <= l:

ValueError: kdims: list length must be between 0 and 0 (inclusive)


````

**Comment by philippjfr** [comment]
Thanks, ``Distribution`` Elements now only support a single value dimension, will have to update the Tutorial.

**Comment by philippjfr** [comment]
Fixed in 00bcb85cf30b3c5bc73eb43bff5de624b64b4b0d, closing.
